src/Configurations/EditorLayer/EditorLayer.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { BrowserRouter } from 'react-router-dom';
+import { IntlProvider } from 'react-intl';
+
+import { Paneset } from '@folio/stripes/components';
+import * as components from '@folio/stripes-acq-components';
+
+import { EditorLayer } from './EditorLayer';
+import { API_BASE, Provider, rest, server } from '../../test/net';
+import { ERROR_RESPONSE } from '../../API/test/setup';
+
+jest.mock('@folio/stripes-acq-components', () => ({
+  ...jest.requireActual('@folio/stripes-acq-components'),
+  useShowCallout: jest.fn(),
+}));
+
+
+const renderLayer = (create) => {
+  const handleClose = () => true;
+
+  return render(
+    <Provider>
+      <BrowserRouter>
+        <IntlProvider locale="en">
+          <Paneset paneTitle="ok">
+            <EditorLayer configurationId="id" create={create} onClose={handleClose} />
+          </Paneset>
+        </IntlProvider>
+      </BrowserRouter>
+
+    </Provider>,
+  );
+};
+
+describe('EditorLayer', () => {
+  it('`useSingleQuery` fires showCalloutHook if any error', async () => {
+    server.use(rest.get(`${API_BASE}/configurations/id`, ERROR_RESPONSE));
+
+    const spy = jest.spyOn(components, 'useShowCallout').mockImplementation(() => jest.fn());
+
+    renderLayer(false);
+
+    expect(spy).toBeCalledTimes(1);
```

Comment:
I'm afraid what you are mocking here, and expecting to be called - is the `useShowCallout` hook itself, but not the function it returns.
The hook will be called regardless. So if you'd not mock the error response, the test still would not fail.